### DISCUSSION TITLE
Tasks for Web Shop Packs and Payment Packs

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#238] [Suggestions]
For this suggestion, I am describing being able to create special yet simple tasks for web shop packs and payment packs. Some task ideas include:

- Task to hide/reveal shop pack on scheduled time. 
- Task to enable/disable servers shop pack can be bought on at scheduled time. 
- That's all I can think of for now lol. Others in the GSA community should definitely submit their own ideas!

**Comment by mvaessen** [comment]
included in #517

**Comment by mvaessen** [comment]
The option to show / hide shop packs have been added. 

I didnt include the `Task to enable/disable servers shop pack can be bought on at scheduled time.`. Feel free to re-submit that idea if think this is still useful. Technically is it quite complicated to implemented that in a way that would make sense.
